00-open-a-shapefile.Rmd

```diff
@@ -159,31 +155,26 @@ aoiBoundary <- readOGR("boundaryFiles/HARV", "HarClip_UTMZ18")
 ```
 
 After loading a new data set we also want to view the attributes of the layer.
-This can be done simple by entering the file name.
+This can be done simple by entering the file name. Alternatively, we can use the `attributes()` function.
```

Comment:
Across lessons do we want to be using notice boxes or the grey bar down left hand side (some older NEON lessons, not sure the code) for Note: or Bonus: material?  
I'm in favor of the grey bar down the side for smaller asides that fit well in the text.  It doesn't break up the flow as much but still sets it apart.  I think the note boxes should be reserved for when we really want to highlight something.